app/audit/models/ClaimARefundAuditModel.scala

```diff
@@ -38,18 +38,20 @@ case class ClaimARefundAuditModel(balanceDetails: Option[BalanceDetails],
   private def getAvailableCredit: Double = {
     balanceDetails.flatMap(_.availableCredit) match {
       case Some(_) => balanceDetails.get.availableCredit.get.toDouble.abs
-      case (_) => 0.00
+      case _ => 0.00
```

Comment:
can represent magic number as val zeroBalance: Double = 0.00 and use zeroBalance instead for example